.vscode/settings.json

```diff
@@ -1,5 +1,10 @@
 {
     "cSpell.words": [
-        "MSAL"
+        "authflow",
+        "Corext",
```

Comment:
Does C need to be upper case here since we use lower case unless it's an acronym like MSAL.

Reply:
Yes it does, because the spelling suggestion was for the actual Enviropnemnt Variable name which can be case sensitive depending on platform. The reason for adding all of these is to avoid accidental spelling or case changes to words in VS Code.